libhmsbeagle/CPU/BeagleCPU4StateSSEImpl.hpp

```diff
@@ -779,8 +781,8 @@ void BeagleCPU4StateSSEImpl<BEAGLE_CPU_4_SSE_DOUBLE>::calcEdgeLogDerivativesPart
         VecUnion vu_m[OFFSET][2];
         SSE_PREFETCH_MATRIX(transMatrix + w, vu_m);
 
-        V_Real * vcl_p = (V_Real *)cl_p;
-        V_Real vwt = VEC_SPLAT(wt[l]);
+//        V_Real * vcl_p = (V_Real *)cl_p;
+//        V_Real vwt = VEC_SPLAT(wt[l]);
```

Comment:
@bredelings Are these lines not used?  Maybe we don't want to change code in non-action files except for that `long long` type change.

Reply:
These lines are not used.  They were giving compiler warnings for unused variables.  In some cases the warnings were getting emitted multiple times because the file was being #include-d multiple times.  I was inspired by Dave Swofford to try to eliminate compiler warnings so that when you get a new warning you can actually notice it.

But this is subjective, so I'd be happy to remove it.  Just let me know.

Reply:
@bredelings Thanks for the explanation.  I like Dave's approach but I didn't write these lines.  Let's keep them untouched for now.

Reply:
OK, will revise.